src/app/descriptors/income/income2dig/index.ts

```diff
@@ -0,0 +1,11 @@
+import { SimpleDescriptor } from '../../../model';
+
+export default new SimpleDescriptor({
+  pathPrefix: 'budget/0000/',
+  titleTemplate: require('../title-template.html'),
+  preTitleTemplate: require('./pretitle-template.html'),
+  amountTemplate: require('../amount-template.html'),
+  subtitleTemplate: require('./subtitle-template.html'),
+  textTemplate: require('../text-template.html'),
+  questions: []
```

Comment:
you can leave here the single question of 'היסטוריה תקציבית' which is relevant for income items too